Kernel_23/test/Kernel_23/include/CGAL/_test_angle.h

```diff
@@ -76,12 +76,10 @@ _test_angle(const R&)
   assert( CGAL::angle( org3, sx, sy, vz) ==  CGAL::ACUTE );
   assert( CGAL::angle( org3, sx, sy, vcoplanar) ==  CGAL::RIGHT );
   assert( CGAL::angle( org3, sx, sy, vmz) ==  CGAL::OBTUSE );
-  assert( CGAL::approximate_angle( sx, org3, sy ) > RT(89.9) );
-  assert( CGAL::approximate_angle( sx-org3, sy-org3 ) > RT(89.9) );
-  assert( CGAL::approximate_angle( sx, org3, sy ) < RT(90.1) );
+  assert( CGAL::approximate_angle( sx, org3, sy ) == RT(90) );
+  assert( CGAL::approximate_angle( sx-org3, sy-org3 ) == RT(90) );
   assert( CGAL::approximate_angle( sx, org3, sx ) == RT(0) );
-  assert( CGAL::approximate_angle( sx, org3, msx ) > RT(179.9) );
-  assert( CGAL::approximate_angle( sx, org3, msx ) < RT(180.1) );
+  assert( CGAL::approximate_angle( sx, org3, msx ) == RT(180) );
```

Comment:
That broke the Kernel_23 testsuite completely. Please, tell me you launched the tests on your machine...
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.14-Ic-58.shtml#Kernel_23

Reply:
Before I had only tested Cartesian, and after fixing Homogenous I didn't test Cartesian again. I better had used ctest.